requirements/common.in

```diff
@@ -42,5 +42,5 @@ jx-mysql
 mo-testing
 
 # Required for Push Health
-mozci==1.4.9
-adr==0.18.7
+mozci
+adr
```

Comment:
I believe we can drop `adr` completely since `mozci` has it as one of its dependencies.